src/util/numeric.cpp

```diff
@@ -161,3 +162,16 @@ bool isBlockInSight(v3s16 blockpos_b, v3f camera_pos, v3f camera_dir,
 
 	return true;
 }
+
+s16 adjustDist(s16 dist, float zoomFov)
```

Comment:
Arguments should be lower case with underscore for coonsistency?
`zoom_fov`

Reply:
agreed.